facets_dive/colab_dive_atlas_demo.ipynb

```diff
@@ -0,0 +1,85 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "colab_dive_atlas_demo.ipynb",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Tkr04WYa8-EF"
+      },
+      "source": [
+        "# Facets Dive demo using sprite atlas in Colab\n",
+        "\n",
+        "* You can directly upload and run this notebook in Colab.  \n",
+        "* The sprite atlas image is passed as a base64 data url. \n",
+        "* Example Colab notebook to create the sprite atlas and metadata can be found [here](https://github.com/jsiddique/facets_example/blob/master/colab_fashion_MNIST_atlas_generation.ipynb).\n",
+        "* To learn more about how sprite atlas file works read [this document](https://github.com/PAIR-code/facets/tree/master/facets_dive)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "uzuyKzjxRdpp"
+      },
+      "source": [
+        "# Display the Dive visualization using atlas and base64 image url.\n",
+        "\n",
+        "import base64\n",
+        "import urllib.request\n",
+        "import os\n",
+        "import pandas as pd\n",
+        "\n",
+        "#Download atlas image\n",
+        "img_url = \"https://raw.githubusercontent.com/jsiddique/facets_example/master/sprite_atlas.png\"\n",
```

Comment:
can you change this to https://storage.googleapis.com/what-if-tool-resources/misc-resources/fmnist_sprite_atlas.png

Reply:
Changed the url.